### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by zhangfc7** [victoresque/pytorch-template#100]
Hi, how can I use more than one loss function?

**Comment by scaomath** [comment]
The simplest way is to write your own:

```python

def loss1(pred, target):
    '''first loss here'''
    return ...

def loss2(pred, target):
    '''second loss here'''
    return ...

def loss(pred, target):
    return a*loss1(pred, target) + b*loss2(pred, target)
```

or something like this, more advanced loss can be implemented through torch's `_WeightedLoss` class template.
